app/components/Project/ProjectFundingStreamForm.tsx

```diff
@@ -0,0 +1,65 @@
+import type { JSONSchema7 } from "json-schema";
+import React, { useMemo } from "react";
+import FormBase from "../Form/FormBase";
+import FormComponentProps from "../Form/FormComponentProps";
+import { ProjectFundingStreamForm_query } from "__generated__/ProjectFundingStreamForm_query.graphql";
+import { graphql } from "react-relay/hooks";
+import { createFragmentContainer } from "react-relay";
+
+interface Props extends FormComponentProps {
+  query: ProjectFundingStreamForm_query;
+}
+
+const uiSchema = {
+  title: "Funding Stream",
+  funding_stream_id: {
+    "ui:placeholder": "Select a Funding Stream",
+    "ui:widget": "select",
+    "ui:col-md": 12,
+    "bcgov:size": "small",
+  },
+};
+
+export const ProjectFundingStreamForm: React.FunctionComponent<Props> = (
+  props
+) => {
+  const fundingStreamList = useMemo(
+    () => props.query.fundingStreams.edges.map((edge) => edge.node.name),
+    [props.query.fundingStreams]
+  );
+  // const fundingStreamDisplayList = useMemo(
+  //   () => props.query.fundingStreams.edges.map((edge) => edge.node.name),
+  //   [props.query.fundingStreams]
+  // );
+  console.log(fundingStreamList);
+
+  var schema: JSONSchema7 = {
+    type: "object",
+    title: "Project Funding Stream",
+    properties: {
+      funding_stream_id: {
+        type: "string",
+        title: "Funding Stream ID",
+        enum: fundingStreamList,
```

Comment:
you may want an `enumNames` here. See the [ProductRowIdField](https://github.com/bcgov/cas-ciip-portal/blob/38e4c5778e1a0112c3eb6cb2def29dd6be67c947/app/containers/Forms/ProductRowIdField.tsx#L41) in CIIP as an example. Typescript might complain as `enumNames` is not part of `JSONSchema7`, so you might need a cast here.
The idea is that we want the id to be saved in the form, but you want to display the funding stream name